client/components/theme-upgrade-modal/index.tsx

```diff
@@ -320,6 +373,15 @@ export const ThemeUpgradeModal = ( {
 		] );
 	};
 
+	const getPersonalPlanFeatureList = () => {
```

Comment:
I think this feature list is a good starting point until we figure this out.

Reply:
I tried that when I was developing the feature, but it breaks the UI (multiline feature that looks bad)